azure/template.json

```diff
@@ -130,6 +130,13 @@
       "metadata": {
         "description": "The port number for the logstash server connection."
       }
+    },
+    "deployFrontDoor": {
+      "type": "string",
+      "defaultValue": "false",
+      "metadata": {
+        "description": "Control Azure Front Door service deployment."
+      }
     }
```

Comment:
Should this be type boolean instead?

Reply:
No, I am using a string type because this information will be passed from a variable in Azure Pipelines